### PR TITLE
RavenDB-13771 Storage summary on the dashboard doesn't include the sy…

### PR DESCRIPTION
**Pull request by Danielle9897**
…stem storage

### CODE REVIEW

**Comment by Danielle9897** [comment]
Closing - I need to fix tests....

File: src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs [hunk 1]
```diff
@@ -97,6 +98,15 @@ private class DatabaseInfoCache
             public DateTime NextDiskSpaceCheck;
         }
 
+        private static readonly SystemInfoCache CachedSystemInfo = new SystemInfoCache();
```
Comment (ayende) [inline src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs]: Why have this?
Note that in some cases, we might have these things come and go dynamically.
I'm also concerned about `MountPoints` being a static `List<T>`, which can cause concurrency issues

File: src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs [hunk 2]
```diff
@@ -211,6 +222,37 @@ public static IEnumerable<AbstractDashboardNotification> FetchDatabasesInfo(Serv
...
+                // 2. Add <system> info 
+                if (isValidFor == null) 
+                {
+                    var currentSystemHash = serverStore._env.CurrentReadTransactionId;
+                    if (currentSystemHash != CachedSystemInfo.Hash || CachedSystemInfo.NextDiskSpaceCheck <  SystemTime.UtcNow)
+                    {
+                        // Update cached value
+                        CachedSystemInfo.Hash = currentSystemHash;
+                        CachedSystemInfo.NextDiskSpaceCheck = SystemTime.UtcNow.AddSeconds(30);
+                        CachedSystemInfo.MountPoints.Clear();
```
Comment (ayende) [inline src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs]: what happens if there is a concurrent read of this at the same time?

Comment (ayende) [reply src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs]: At a minimum, you need to replace the whole object in one go

Comment (Danielle9897) [reply src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs]: Code refactored - See the second commit

File: src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs]: Code refactored - See the second commit

**Pull request merged**